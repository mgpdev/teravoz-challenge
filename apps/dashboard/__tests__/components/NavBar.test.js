import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

// Component to be tested
import NavBar from "./../../src/components/NavBar";

describe("<NavBar />", () => {
  describe("render()", () => {
    test("renders the component", () => {
      const wrapper = shallow(<NavBar />);

      const component = wrapper.html();

      expect(component).toMatchSnapshot();
    });
  });
});
