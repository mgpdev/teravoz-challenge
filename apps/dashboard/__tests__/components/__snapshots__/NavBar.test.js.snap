// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> render() renders the component 1`] = `"<div><header class=\\"MuiPaper-root-9 MuiPaper-elevation4-15 MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-7\\"><div class=\\"MuiToolbar-root-36 MuiToolbar-regular-38 MuiToolbar-gutters-37\\"><h2 class=\\"MuiTypography-root-40 MuiTypography-title-46 MuiTypography-colorInherit-59\\">Teravoz Dashboard</h2></div></header></div>"`;
